soccer/gameplay/single_robot_composite_behavior.py

```diff
@@ -10,6 +10,24 @@
 # Note: The method-resolution-order in python dictates that as we have it listed below, SingleRobotBehavior methods take precedence over CompositeBehavior methods
 class SingleRobotCompositeBehavior(single_robot_behavior.SingleRobotBehavior,
                                    composite_behavior.CompositeBehavior):
+    #If the behavior should not be restarted when it is assigned to a different robot, then restart_behavior_on_switch_robot must be false
+    def __init__(self,
+                 continuous=False,
+                 restart_behavior_on_switch_robot=True):
```

Comment:
Actually, on further thought, it might be better to take in a function here, so that plays can fine tune if they want to be reset or not. I'm not sure how this would work though (run the function every tick, or should we run it whenever we have a robot change). This needs to be thought out more, but I think it would be great to have it in here.